js/commonMenu.js

```diff
@@ -314,90 +306,3 @@ module.exports.aboutBraveMenuItem = () => {
     }
   }
 }
-
-module.exports.buildBraveryMenu = function (settings, init) {
```

Comment:
I think we should leave this in for now, at least in the app menu. Otherwise users have no way to change their global settings. Ex: a user who has noscript on will have no way to turn it off for every site except to go to the site and manually turn off each one.